<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TardisManagerScreen</class>
 <widget class="QDialog" name="TardisManagerScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>961</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DM: ACK - TARDIS Manager</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>70</y>
     <width>171</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DM-Typeface</family>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border: none;
    border-image: url(:/resources/button_normal.png) 0 0 0 0 stretch stretch;
    width: 200px;
    height: 100px;
	color: #3c3c3c;
padding-bottom: -4px;
}

QPushButton:hover {
    border-image: url(:/resources/button_hovered.png) 0 0 0 0 stretch stretch;
color: black;
padding-bottom: -4px;
}

QPushButton:pressed {
  border-image: url(:/resources/button_pressed.png) 0 0 0 0 stretch stretch;
color: black;
padding-bottom: -8px;
}</string>
   </property>
   <property name="text">
    <string>Add Exterior</string>
   </property>
  </widget>
  <widget class="QPushButton" name="interiorButton">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>110</y>
     <width>171</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DM-Typeface</family>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border: none;
    border-image: url(:/resources/button_normal.png) 0 0 0 0 stretch stretch;
    width: 200px;
    height: 100px;
	color: #3c3c3c;
padding-bottom: -4px;
}

QPushButton:hover {
    border-image: url(:/resources/button_hovered.png) 0 0 0 0 stretch stretch;
color: black;
padding-bottom: -4px;
}

QPushButton:pressed {
  border-image: url(:/resources/button_pressed.png) 0 0 0 0 stretch stretch;
color: black;
padding-bottom: -8px;
}</string>
   </property>
   <property name="text">
    <string>Add Interior</string>
   </property>
  </widget>
  <widget class="QPushButton" name="layerButton">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>150</y>
     <width>171</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DM-Typeface</family>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border: none;
    border-image: url(:/resources/button_normal.png) 0 0 0 0 stretch stretch;
    width: 200px;
    height: 100px;
	color: #3c3c3c;
padding-bottom: -4px;
}

QPushButton:hover {
    border-image: url(:/resources/button_hovered.png) 0 0 0 0 stretch stretch;
color: black;
padding-bottom: -4px;
}

QPushButton:pressed {
  border-image: url(:/resources/button_pressed.png) 0 0 0 0 stretch stretch;
color: black;
padding-bottom: -8px;
}</string>
   </property>
   <property name="text">
    <string>Add Layer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="layerCheckButton">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>190</y>
     <width>171</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DM-Typeface</family>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border: none;
    border-image: url(:/resources/button_normal.png) 0 0 0 0 stretch stretch;
    width: 200px;
    height: 100px;
	color: #3c3c3c;
padding-bottom: -4px;
}

QPushButton:hover {
    border-image: url(:/resources/button_hovered.png) 0 0 0 0 stretch stretch;
color: black;
padding-bottom: -4px;
}

QPushButton:pressed {
  border-image: url(:/resources/button_pressed.png) 0 0 0 0 stretch stretch;
color: black;
padding-bottom: -8px;
}</string>
   </property>
   <property name="text">
    <string>Add Layer Check</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>741</width>
     <height>521</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>741</width>
      <height>521</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>400</height>
     </size>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>741</width>
       <height>511</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout"/>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>528</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DM-Typeface</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border: none;
    border-image: url(:/resources/button_normal.png) 0 0 0 0 stretch stretch;
    width: 200px;
    height: 100px;
	color: #3c3c3c;
padding-bottom: -4px;
}

QPushButton:hover {
    border-image: url(:/resources/button_hovered.png) 0 0 0 0 stretch stretch;
color: black;
padding-bottom: -4px;
}

QPushButton:pressed {
  border-image: url(:/resources/button_pressed.png) 0 0 0 0 stretch stretch;
color: black;
padding-bottom: -8px;
}</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="tardisManagerLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>921</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DM-Typeface</family>
     <pointsize>22</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TARDIS MANAGER</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>971</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>20</width>
     <height>541</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
